.github/workflows/tests.yaml

```diff
@@ -39,52 +41,77 @@ jobs:
+    - name: Build AuthFoundation
+      run: |
+        set -o pipefail
+        env NSUnbufferedIO=YES xcodebuild build \
+          -derivedDataPath ../Build/DerivedData \
+          -clonedSourcePackagesDirPath ../Build/ClonedSources \
+          -scheme AuthFoundation \
+          -sdk iphonesimulator \
+          -destination "$iOS_DESTINATION" | xcpretty
+    - name: Build OktaOAuth2
+      run: |
+        set -o pipefail
+        env NSUnbufferedIO=YES xcodebuild build \
+          -derivedDataPath ../Build/DerivedData \
+          -clonedSourcePackagesDirPath ../Build/ClonedSources \
+          -scheme OktaOAuth2 \
+          -sdk iphonesimulator \
+          -destination "$iOS_DESTINATION" | xcpretty
+    - name: Build WebAuthenticationUI
+      run: |
+        set -o pipefail
+        env NSUnbufferedIO=YES xcodebuild build \
+          -derivedDataPath ../Build/DerivedData \
+          -clonedSourcePackagesDirPath ../Build/ClonedSources \
+          -scheme WebAuthenticationUI \
+          -sdk iphonesimulator \
+          -destination "$iOS_DESTINATION" | xcpretty
+    - name: Prepare build artifacts
+      run: |
+        tar -czf ../iOS-Build.tgz ../Build
     - name: Test AuthFoundation
       run: |
         set -o pipefail
         env NSUnbufferedIO=YES xcodebuild test \
           -derivedDataPath ../Build/DerivedData \
           -clonedSourcePackagesDirPath ../Build/ClonedSources \
-          -resultBundlePath ../TestResults/AuthFoundation.xcresult \
           -scheme AuthFoundation \
           -sdk iphonesimulator \
-          -destination "platform=iOS Simulator,OS=latest,name=iPhone 12" | xcpretty
+          -destination "$iOS_DESTINATION" | xcpretty
     - name: Test OktaOAuth2
       run: |
         set -o pipefail
         env NSUnbufferedIO=YES xcodebuild test \
           -derivedDataPath ../Build/DerivedData \
           -clonedSourcePackagesDirPath ../Build/ClonedSources \
-          -resultBundlePath ../TestResults/OktaOAuth2.xcresult \
           -scheme OktaOAuth2 \
           -sdk iphonesimulator \
-          -destination "platform=iOS Simulator,OS=latest,name=iPhone 12" | xcpretty
+          -destination "$iOS_DESTINATION" | xcpretty
     - name: Test WebAuthenticationUI
       run: |
         set -o pipefail
         env NSUnbufferedIO=YES xcodebuild test \
           -derivedDataPath ../Build/DerivedData \
           -clonedSourcePackagesDirPath ../Build/ClonedSources \
-          -resultBundlePath ../TestResults/WebAuthenticationUI.xcresult \
           -scheme WebAuthenticationUI \
           -sdk iphonesimulator \
-          -destination "platform=iOS Simulator,OS=latest,name=iPhone 12" | xcpretty
+          -destination "$iOS_DESTINATION" | xcpretty
     - name: Upload Test Results
       uses: actions/upload-artifact@v3
       with:
-        path: ~/TestResults/*.xcresult
-        name: XcodeBuild-Results
-        if-no-files-found: warn
+        path: ../iOS-Build.tgz
+        name: iOS-Build
```

Comment:
This is no longer uploading test results

Reply:
Yes, the test result post-processing was error prone and added several minutes to the build time. I didn't feel like it was adding that much value, so I'm taking it out.